Formula/wiremock-standalone.rb

```diff
@@ -1,8 +1,9 @@
 class WiremockStandalone < Formula
-  desc "Simulator for HTTP-based APIs"
+  desc "A tool for mocking HTTP services"
```

Comment:
I think I like our current description better… “tool” is very generic, “mocking” is not so informative as it's already in the formula name.

Reply:
Sure, I can make that change, the description is adopted from their github tagline.